components/Footer.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from "styled-components/native";
+import { View, Text, Linking } from 'react-native';
+
+
+export const Footer = () => {
+    return (
+        <FooterContainer>
+            <FooterText>To make someone else's day, please click the link below and write about somehing that made you smile today</FooterText>
+            <FooterLink onPress={() => Linking.openURL('https://spring2022-happy-tweets.netlify.app/')}>Happy Tweets</FooterLink>
```

Comment:
To link this project into another project of yours is just brilliant. Makes this weeks project a little bit more dynamic and interesting as well as the viewer can get 2 of your built masterpieces:-)
Great work that you found the React Native structure of adding an URL/Link